cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/handler/sync/AdminRequest.java

```diff
@@ -129,12 +129,20 @@ protected String processChangeExecutionConcurrencyRequest() {
       LOG.info("Intra-broker partition movement concurrency is set to: {} by user.", concurrentIntraBrokerPartitionMovements);
     }
     // 3. Change leadership concurrency for all brokers.
```

Comment:
Since now we have the cluster concurrency, we should change this to "Change cluster leadership concurrency". And rename setExecutionConcurrencyForAllBrokers on L134 to something like setClusterExecutionConcurrency